Packs/CommonScripts/Scripts/ShowLocationOnMap/ShowLocationOnMap.py

```diff
@@ -1,6 +1,8 @@
 from CommonServerPython import *
 
-loc = demisto.get(demisto.args()['indicator'], "CustomFields.geolocation").strip()
+loc = demisto.get(demisto.args().get('indicator') or {}, "CustomFields.geolocation")
+if loc and isinstance(loc, str):
+    loc = loc.strip()
 err_msg = "No location data was available"
 long_lat_regex = re.compile(
```

Comment:
The rest of the code should be under this if as well.

Reply:
Done, I hope that i have done what you meant